internal_task_pollers.go

```diff
@@ -217,10 +217,10 @@ func (wtp *workflowTaskPoller) poll() (*workflowTask, error) {
 		Identity: common.StringPtr(wtp.identity),
 	}
 
-	ctx, cancel := newTChannelContext(tchanTimeout(pollTaskServiceTimeOut), tchanRetryOption(retryNeverOptions))
+	tchCtx, cancel := newTChannelContext(context.Background(), tchanTimeout(pollTaskServiceTimeOut), tchanRetryOption(retryNeverOptions))
```

Comment:
Should we also accept context from user for poll requests?  This will make graceful shutdown easier in the future.

Reply:
We can discuss that separately, but it is not in the scope of this change.